bin/natalie

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env ruby
 
+if RUBY_ENGINE != 'natalie'
+  require 'bundler/inline'
+
+  gemfile do
+    source 'https://rubygems.org'
+    gem 'debug', '1.6.2'
+  end
+end
+
```

Comment:
I think this should go in `Gemfile` with the other development dependencies?

Reply:
Actually, I just realized we removed the need for Bundler when we removed RubyParser... I'm hesitant to add it back in. Do you think this could be made optional somehow?

Reply:
The Gemfile is not loaded when running bin/natalie... However the gem seems to make problems on CI anyway 😢

Reply:
> Actually, I just realized we removed the need for Bundler when we removed RubyParser... I'm hesitant to add it back in. Do you think this could be made optional somehow?

We could hide it behind a DEBUG environment variable or something?